### PR TITLE
Fix #15: Add release anchor links.

### LINKED ISSUE #15: Add anchors for permalinks to release infos
Bob wants to send a link to Alice so she can check that the users of Firefox 51 will receive the hsts-pinning system-addon :)
Ref https://bugzilla.mozilla.org/show_bug.cgi?id=1335224#c13

### PR DESCRIPTION
**Pull request by n1k0**
Fix #15.

### CODE REVIEW

**Comment by n1k0** [comment]
Deployed to gh-pages ([test](https://leplatrem.github.io/systemaddons-versions/#eb1e4bb4cb16001296343af1a44d112b)).

**Comment by leplatrem** [comment]
Unfortunately, it does not seem to work with the async loading :( 

Eg. for the use-case described in the issue: open this in a new tab https://leplatrem.github.io/systemaddons-versions/#0954c5f32206570c1c903c97321332eb

**Comment by n1k0** [comment]
Crap. I'll try to find a better way.

**Comment by n1k0** [comment]
@leplatrem fixed in https://github.com/leplatrem/systemaddons-versions/pull/16/commits/dfd41c08f22c3f64d136a5d45cbab724b8e2021d

File: ui/index.html [hunk 1]
```diff
@@ -10,6 +10,15 @@
...
   <body>
     <script src="app.js"></script>
-    <script>Elm.Main.fullscreen()</script>
+    <script>
+    const hash = location.hash;
+    location.hash = "";
+    const app = Elm.Main.fullscreen();
+    app.ports.onFetched.subscribe(() => {
+      if (hash) {
+        setTimeout(() => location.hash = hash, 100);
```
Comment (n1k0) [inline ui/index.html]: These 100ms is the amount of time required for the browser to fully render the list of releases after they've been received. We can't precisely know this value, so I fiddled a little and came with `100`.

**Review by leplatrem — APPROVED** [review]
Works here :) 

Not a lot of comments in the code though ;)

**Pull request merged**